pkgs/servers/klipper/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python2
+}: stdenv.mkDerivation rec {
+  name = "klipper";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "KevinOConnor";
+    repo = "klipper";
+    rev = "v${version}";
+    sha256 = "1ijy2ij9yii5hms10914i614wkjpsy0k4rbgnm6l594gphivdfm7";
+  } + "/klippy";
+
+  nativeBuildInputs = [ (python2.withPackages (p: with p; [ cffi pyserial greenlet jinja2 ])) ];
```

Comment:
eww - is there an upstream bug report tracking to port this to python 3 you could link to?

Reply:
They're slowly working towards it: https://github.com/KevinOConnor/klipper/pull/3278

I'll link it in the code

Reply:
Done